### PR TITLE
Docs: add Zoom to Reporting docs

### PR DESCRIPTION
**Pull request by mjseaman**
Add a brief description of the Zoom option for Reports.

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/dashboards/create-reports/index.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ Only organization administrators can create reports by default. You can customiz
    - If you selected the PDF format option:
      - Select an orientation for the report: **Portrait** or **Landscape**.
      - Select a layout for the generated report: **Simple** or **Grid**. The simple layout renders each panel as full-width across the PDF. The grid layout renders the PDF with the same panel arrangement and width as the source dashboard.
+     - Select a zoom level for the report. Zoom in to enlarge text in your PDF, or zoom out to see more data (like table columns) per panel.
```
Comment (imatwawana) [inline docs/sources/dashboards/create-reports/index.md:132]:
```suggestion
     - Select a **Zoom** level for the report. Zoom in to enlarge text in your PDF, or zoom out to see more data (like table columns) per panel.
```

**Review by imatwawana — COMMENTED** [review]
Wording looks fine to me. Is this a new feature or just missing documentation for an old one? If it's been around for a while, we should backport the change (only as far back as 10.4 though).

**Review by imatwawana — APPROVED** [review]

File: docs/sources/dashboards/create-reports/index.md [hunk 2]
```diff
@@ -129,6 +129,7 @@ Only organization administrators can create reports by default. You can customiz
    - If you selected the PDF format option:
      - Select an orientation for the report: **Portrait** or **Landscape**.
      - Select a layout for the generated report: **Simple** or **Grid**. The simple layout renders each panel as full-width across the PDF. The grid layout renders the PDF with the same panel arrangement and width as the source dashboard.
+     - Select a **Zoom** level for the report. Zoom in to enlarge text in your PDF, or zoom out to see more data (like table columns) per panel.
```
Comment (imatwawana) [inline docs/sources/dashboards/create-reports/index.md:132]:
```suggestion
     - Select a zoom level for the report. Zoom in to enlarge text in your PDF, or zoom out to see more data (like table columns) per panel.
```

**Comment by diegommm** [comment]
Hi! Greetings from the Release Guild. This is a friendly reminder that the Steady [11.2.x Release Branch](https://github.com/grafana/grafana/tree/v11.2.x) was cut this week, and that it includes the changes committed up to https://github.com/grafana/grafana/commit/b4126d3bce3627e9460e2db0e870c8d984be8e84. This PR was merged after it, so we wanted to let you know that you can add the `backport v11.2.x` label to to backport it and make it part of the next 11.2.x Patch Release, just in case you may have missed the news.

Thank you!

**Pull request merged**